babel-config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ["@babel/react", "@babel/env"]
+};
```

Comment:
This is the same config that is in all the package-specific `babelrc` files, but it's not working for test compilation.